src/constants/mimeTypes.ts

```diff
@@ -15,26 +15,26 @@ const VIDEO = {
   WEBM: 'video/webm',
 };
 
-const AUDIO = {
+const Audio = {
   MPEG: 'audio/mpeg',
   MP3: 'audio/mp3',
   WAV: 'audio/wav',
   WAV_Chrome: 'audio/x-wav',
 };
 
-const PDF = 'application/pdf';
+const Pdf = 'application/pdf';
 
-const ZIP = 'application/zip';
+const Zip = 'application/zip';
```

Comment:
Nitpick: should these stay uppercase since they are actual constants? Or maybe should they be packed into an `Application` enum, with `PDF` and `ZIP` as keys (this would be consistent with Audio and Video for instance)?

Reply:
I will change them back to UPPER_CASE since we want to express our own model and we do not really follow the mimetype containers here.